examples/material_gallery/lib/gallery/home.dart

```diff
@@ -106,51 +73,51 @@ class GalleryHomeState extends State<GalleryHome> {
             leading: new Icon(icon: Icons.star),
             title: new Text('Demos'),
             children: <Widget>[
-              new GalleryItem(title: 'Weather', builder: () => new WeatherDemo()),
-              new GalleryItem(title: 'Fitness', builder: () => new FitnessDemo()),
-              new GalleryItem(title: 'Fancy lines', builder: () => new DrawingDemo()),
-              new GalleryItem(title: 'Flexible space toolbar', builder: () => new FlexibleSpaceDemo()),
-              new GalleryItem(title: 'Floating action button', builder: () => new TabsFabDemo()),
+              new GalleryItem(title: 'Weather', routeName: WeatherDemo.routeName),
```

Comment:
This array could also be generated from a global list of kDeoms = [new WeatherDemo(), ...] by:
()=> new GalleryItem(demo.title, builder: () => demo);